armi/bookkeeping/visualization/entryPoint.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2020 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import re
+import sys
+import pathlib
+
+from armi import runLog
+from armi.cli import entryPoint
+
+
+class VisFileEntryPoint(entryPoint.EntryPoint):
+    """
+    Create visualization files from database files.
+    """
+
+    name = "vis-file"
```

Comment:
is this using `vis` as the verb: _to visualize_ ? I guess that makes sense. At first I read it as a noun

Reply:
i think of it as shorthand for "visualization file", but that isnt really a verb i suppose. "make-vis-file" is too long. "vis-file" just seems so _right_